app/src/main/res/xml/actions.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version ="1.0" encoding ="utf-8"?>
```

Comment:
Does it make sense to add a reference to the app actions documentation if someone else reads this code?